core/commands/storage/upload/sessions/renter_sessions.go

```diff
@@ -156,6 +157,15 @@ func (rs *RenterSession) enterState(e *fsm.Event) {
 		rs.Cancel()
 	case RssCompleteStatus:
 		rs.Cancel()
+		// call `btfs storage contracts sync renter`
+		go func() {
+			sh := shell.NewLocalShell()
+			ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+			defer cancel()
+			if err := sh.Request("storage/contracts/sync/renter").Exec(ctx, nil); err != nil {
```

Comment:
Expensive approach.. can you call router sync helpers instead?

Reply:
do u mean that call the function inside process?
will do

Reply:
yes